packages/dashboard/package.json

```diff
@@ -22,15 +23,18 @@
     "@servie/events": "^3.0.0",
     "@types/color": "^3.0.1",
     "async-call-rpc": "^5.1.0",
+    "bignumber.js": "^9.0.1",
```

Comment:
Is it ok? @guanbinrui

Reply:
I think there is no problem to add this package.